cmd/console/console.go

```diff
@@ -210,9 +210,10 @@ func modifySshdConfig() error {
 	return ioutil.WriteFile("/etc/ssh/sshd_config", []byte(sshdConfigString), 0644)
 }
 
+// https://www.freedesktop.org/software/systemd/man/os-release.html
 func writeOsRelease() error {
 	idLike := "busybox"
-	if osRelease, err := ioutil.ReadFile("/etc/os-release"); err == nil {
+	if osRelease, err := ioutil.ReadFile("/usr/lib/os-release"); err == nil {
```

Comment:
I feel bad changing `/usr/lib` as that is supposed to be read only.  According to the fake spec `/etc/os-release` should be read by application first.  So I'm guessing Docker is doing the wrong thing here if they complain `/usr/lib/os-release` isn't there if `/etc/os-release` is.